crates/router/src/connector/stripe/transformers.rs

```diff
@@ -1199,7 +1154,6 @@ fn get_bank_debit_data(
 }
 
 fn create_stripe_payment_method(
```

Comment:
Let's also implement individual try_from for each payment_method_data type instead of handling in a single method.Take reference from Adyen transformer.

Further optimization can be done here, we can discuss this offline